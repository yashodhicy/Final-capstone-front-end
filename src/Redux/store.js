import { configureStore } from '@reduxjs/toolkit';
import HouseSlice from './HouseSlice';
import reducer from './reservation/reducer';


const store = configureStore({
  reducer: {
    Houses: HouseSlice,
    reservations: reducer
  },
});

export default store;
