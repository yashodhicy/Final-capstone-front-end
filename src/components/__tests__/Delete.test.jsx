import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../../Redux/store';
import DeleteHouse from '../Delete' ;

describe('Test Delete Component', () => {
    test ('it renders correctly', () => {
        const tree = renderer
        .create(
            <Provider store={store}>
                <Router>
                    <DeleteHouse />
                </Router>
            </Provider>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    })
});
