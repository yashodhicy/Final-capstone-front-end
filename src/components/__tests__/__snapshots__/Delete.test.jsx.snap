// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Delete Component it renders correctly 1`] = `
<div>
  <section
    className="houses"
  >
    <div
      className="header"
    >
      <h2>
        Need to delete your house?
      </h2>
      <p>
        After deleting others can not reserve your house
      </p>
    </div>
    <p>
      You have no houses yet. Add a new house.
    </p>
  </section>
</div>
`;
